insta-ui/package.json

```diff
@@ -28,6 +28,7 @@
     "next": "14.2.13",
     "react": "^18",
     "react-dom": "^18",
+    "react-icons": "^5.3.0",
```

Comment:
we've imported the font-awesome so you can use that one to reduce bundle size!

Reply:
Resolved